dev/README.md

```diff
@@ -0,0 +1,24 @@
+# Dev tools for the MLflow website
+
+## Local generation of release notes formatting
+
+If you would like to generate the required links for all release notes in the `website/releases/` directory, you can utilize the 
+dev script located at `dev/local-format.js` as follows:
+
+From the root of the repository, simply run:
+
+```shell
+node dev/local-format.js
```

Comment:
we could also consider adding this script to `package.json` (https://github.com/mlflow/mlflow-website/blob/main/website/package.json#L5-L18), then people can run it from the `website` directory with `yarn local-format`. the entry would look something like:

```json
"scripts": {
  ...
  "local-format": "node ../dev/local-format.js",
  ...
},
```

Reply:
This is a fantastic idea. Migrated and made into a yarn script! (updated dev guide with the script target as well)